### PR TITLE
Refractoring

### PR DESCRIPTION
**Pull request by quarno**
changed `#!/bin/bash` to `#!/usr/bin/env bash` (see [here](https://github.com/dylanaraps/pure-bash-bible#obsolete-syntax) why)

Using cat instead of echo to write files with more than one line

### CODE REVIEW

**Comment by quarno** [comment]
> We shouldn't "force" users to run bash with the resulting start.sh and start_autorestart.sh scripts, since they are basic enough to work with any shell (even _install.sh works with dash and zsh, only _install_mods.sh fails with zsh, but for one time run I don't think it's necessarily an issue to use bash).

Ok, I should have done two separate commits. Is the cat usage good though?

**Comment by Lartza** [comment]
Doesn't github tell you if you just commit more it gets added to this PR? :) That's how you can just fix it with a second commit. But yes everything else seemed in good shape

**Comment by quarno** [comment]
Ok I reverted the changes involving the first line of the scripts

File: server/_install.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (Lartza) [inline server/_install.sh]: I meant that this can stay as #!/usr/bin/env bash

File: server/_install.sh [hunk 2]
```diff
@@ -22,15 +22,15 @@ else
 fi
 
 cat > start.sh <<EOF
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (Lartza) [inline server/_install.sh]: This is the one I wanted changing, but it should be pointing to **sh** not bash, env or not shouldn't matter necessarily.

File: server/_install.sh [hunk 3]
```diff
@@ -22,15 +22,15 @@ else
...
 
 java -Xms2499M -Xmx2500M -jar fabric-server-launch.jar nogui
 
 read -n1 -r -p "Press any key to continue..."
 EOF
 
 cat > start_autorestart.sh <<EOF
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (Lartza) [inline server/_install.sh]: **sh** here too, env or not

File: server/_install.sh (hunk 1 above)
Comment (Lartza) [reply server/_install.sh]: (Although sh works with this file too afaict, dash and zsh seemed to work)

File: server/_install_mods.sh [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (Lartza) [inline server/_install_mods.sh]: This can also stay as #!/usr/bin/env bash

**Comment by quarno** [comment]
Done, sorry for my misunderstanding

**Comment by calloatti** [comment]
Merged. Please check that everything is OK, since I know nothing about Unix style batch files. Thank you both.

**Pull request merged**